src/Sulu/Bundle/AdminBundle/Resources/js/components/ItemSelection/header.scss

```diff
@@ -0,0 +1,28 @@
+@import '../../containers/Application/colors.scss';
+
+$labelTextColor: $scorpion;
+$contentBackgroundColor: $alto;
+
+.header {
+    display: flex;
+    justify-content: space-between;
+    height: 40px;
+    overflow: hidden;
+    border-radius: 3px 3px 0 0;
```

Comment:
That's something i don't understand... Why is the `border-radius` just not always 3px everywhere?

Reply:
According to the design the header has a different `border-radius` if the list underneath is or is not empty